tests/e2e/test_slack_post.py

```diff
@@ -3,13 +3,15 @@
 import unittest
 
 import certifi
+import pytest
 from slack import WebClient
 from slack.errors import SlackApiError
 
 from src.fetch.transfer_file import post_to_slack
 
 
 class TestSlackPost(unittest.TestCase):
+    @pytest.mark.skip(reason="Too many accidental slack posts - need to mock this!")
```

Comment:
A nice to have would be to link to the issue which implements this so we know there's already some info about it.

Reply:
Sure thing. Will add.